packages/cli/src/commands/init/init.js

```diff
@@ -22,12 +24,55 @@ import installPods from '../../tools/installPods';
 import {processTemplateName} from './templateName';
 import banner from './banner';
 import {getLoader} from '../../tools/loader';
+import {CLIError} from '@react-native-community/cli-tools';
+
+const DEFAULT_VERSION = 'latest';
 
 type Options = {|
   template?: string,
   npm?: boolean,
+  directory?: string,
 |};
 
+function doesDirectoryExist(dir: string) {
+  return fs.existsSync(dir);
+}
+
+function getProjectDirectory({projectName, directory}): string {
+  return path.relative(process.cwd(), directory || projectName);
+}
+
+async function setProjectDirectory(directory) {
+  const directoryExists = doesDirectoryExist(directory);
+  if (directoryExists) {
+    const {shouldReplaceprojectDirectory} = await inquirer.prompt([
```

Comment:
Can we default to the `No` option here? We don't want users to overwrite files by a mistake.

Reply:
Makes sense, fixed

Reply:
Default option is still `Yes`

<img width="754" alt="Screenshot 2019-05-17 at 17 03 21" src="https://user-images.githubusercontent.com/9092510/57937195-b873d480-78c5-11e9-9d70-09c02c0f4984.png">

Reply:
fixd, pushed it to wrong branch